### PR TITLE
upgrade runner version

### PR DESCRIPTION
**Pull request by TDH55**
Mobile previewer was crashing for developers trying to work with file and image data types. I updated the runner version to match the one used in the template app repo.

### CODE REVIEW

**Review by paulopompeu — APPROVED** [review]

**Comment by TDH55** [comment]
[Image uploads from mobile not working correctly with 3rd party components](https://trello.com/c/MTx3cmoF/450-image-uploads-from-mobile-not-working-correctly-with-3rd-party-components)

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@protonapp/material-components": "https://component-marketplace-prod.s3.amazonaws.com/@protonapp/material-components/0.8.17.tar",
-    "@protonapp/proton-runner": "^0.2.30",
+    "@protonapp/proton-runner": "^0.4.71",
```
Comment (guislain-adalo) [inline package.json:15]: That's a pretty significant bump. Are there things you are worried could break?

**Review by guislain-adalo — COMMENTED** [review]
LGTM but I'm wondering if this could come with some surprises. I don't know this area well so my worries might be unjustified. What do you think?

File: package.json (hunk 1 above)
Comment (TDH55) [reply package.json:15]: There shouldn't be, I'm not aware of any breaking changes we've made to runner and this simply brings it in line with the rest of our code base. This is also solely developer facing and won't effect makers/end users if we do need to revert for any reason.

At some point we should go through and make sure all dependencies in `mobile-previewer` match the versions in `template-app` so we can be sure that component developer's are testing their native components on an up to date version of Adalo

Comment (guislain-adalo) [reply package.json:15]: Sounds good. Thanks

🛳  it

**Pull request merged**